src/__tests__/lib/containers/provenance/ProvenanceGraph.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { render, screen, waitFor, within } from '@testing-library/react'
+import React from 'react'
+import {
+  ProvenanceGraph,
+  ProvenanceProps,
+} from '../../../../lib/containers/provenance/ProvenanceGraph'
+import { createWrapper } from '../../../../lib/testutils/TestingLibraryUtils'
+import { ACTIVITY_FOR_ENTITY } from '../../../../lib/utils/APIConstants'
+import {
+  BackendDestinationEnum,
+  getEndpoint,
+} from '../../../../lib/utils/functions/getEndpoint'
+import { mockActivity } from '../../../../mocks/provenance/mockActivity'
+import mockFileEntityData from '../../../../mocks/entity/mockFileEntity'
+import { rest, server } from '../../../../mocks/msw/server'
+import { SynapseApiResponse } from '../../../../mocks/msw/handlers'
+import { Activity } from '../../../../lib/utils/synapseTypes/Provenance/Provenance'
+import { MOCK_TABLE_ENTITY_ID } from '../../../../mocks/entity/mockEntity'
+
+function renderComponent(props: ProvenanceProps) {
+  return render(<ProvenanceGraph {...props} />, { wrapper: createWrapper() })
+}
+
+describe('ProvenanceGraph', () => {
```

Comment:
That's the big question, isn't it.  every time I think it's stable I realized that I needed to either optimize something or support an additional feature which would break any existing tests.  We may want to wait for design (for the ARK portal) to see if any big changes are needed to the way it works today.